exercises/yacht/canonical-data.json

```diff
@@ -0,0 +1,261 @@
+{
+    "exercise" : "yacht",
+    "version" : "1.0.0",
+    "comments" : [
+        "The dice are represented always as a list of exactly five integers",
+        "with values beteen 1 and 6 inclusive. The category is an integer.",
+        "the numbers 0 to 5 represent the categories 'ones' to 'sixes'",
+        "Then 6 is full house,",
+        "     7 is four of a kind",
+        "     8 is a little straight 1-5",
+        "     9 is b big straight 2-6",
+        "    10 is Choise, sometimes called Chance",
+        "    11 is Yacht, or five of a kind"
+    ],
+    "cases": [
+	{
+	
+        "description": "Yacht",
+        "property":    "score",
+        "input":       {
+		    "dice":    [5, 5, 5, 5, 5],
+		    "category":11
+        },
+		"expected":    50
```

Comment:
I have a hard time reading this `expected` line since it is not indented to the same column as other keys on its level (`description`, `property`, `input`). I would request that `expected`'s indentation be changed to be the same column, for the purpose of readability.

I have a similar hard time reading further cases below in this file where `description` and `expected` are indented to some column but `property` and `input` are indented to some other column, and I make the same request (change them so that they indent to the same column).